com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 ## [2.2.0-exp.3] - 2021-01-22
 ### Changes
 - Minimum Unity version is 2019.4
```

Comment:
Since we're here:
`- Changed the Unity Editor minimum compatible version to 2019.4.`